.github/workflows/scheduled.yml

```
@@ -396,3 +396,72 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+
+  presto-java-aggregation-fuzzer-run:
+    runs-on: 8-core
+    container: ghcr.io/facebookincubator/velox-dev:presto-java
+    timeout-minutes: 120
+    if: "$${{ github.event_name != 'pull_request' }}"
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/.ccache/"
+      CCACHE_BASEDIR: "${{ github.workspace }}"
+      LINUX_DISTRO: "centos"
+    steps:
+
+      - name: "Restore ccache"
+        uses: assignUser/stash/restore@v1
+        with:
+          path: "${{ env.CCACHE_DIR }}"
+          key: ccache-fuzzer
+
+      - name: "Checkout Repo"
+        uses: actions/checkout@v4
+        with:
+          path: velox
+          submodules: 'recursive'
+          ref: "${{ inputs.ref }}"
+
+      - name: "Install dependencies"
+        run: |
+          cd  velox
+          source ./scripts/setup-ubuntu.sh
+          ccache -vsz
```

```suggestion
      - name: "Zero Ccache Statistics"
        run:  ccache -sz
```
The deps are already installed in the container. Centos 8 ccache doens't support -v